change/@fluentui-react-native-e2e-testing-93969873-950e-4c8e-addd-f384491bc7c1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
I think these should be "minor" instead of "patch". I don't think adding new tests warrants a patch bump. Changing something bigger to the infrastructure would be a patch.

Reply:
That makes sense. Will change.